github_connector_odoo/models/github_organization.py

```diff
@@ -24,11 +24,12 @@ class GithubOrganization(models.Model):
 
     fetch_branches = fields.Boolean(
         string="Fetch all branches",
-        help="If checked, Fetch all repo branches even " "if no series are defined",
+        help="If checked, Fetch all repo branches even if no series are defined",
     )
 
     odoo_versions = fields.Text(
-        default="5.0, 6.0, 6.1, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0, 13.0, 14.0, 15.0, 16.0",
+        default="5.0, 5, 6.0, 6.1, 6, 7.0, 7, 8.0, 8, 9.0, 9, 10.0, 10, 11.0, 11, "
```

Comment:
Why Text?
And Odoo versions are global, no need to keep them for every organisation. Keeping them in system parameters looks much more reasonable.

Reply:
resolved